packages/react-native-otlp/src/index.ts

```diff
@@ -0,0 +1,88 @@
+"use strict";
+import {Platform, NativeModules} from "react-native";
+
+import {createFalsePromise} from "./utils";
+
+interface CustomExporterConfig {
+  endpoint: string;
+  header?: {key: string; token: string};
+  timeout?: number;
+}
+
+interface OTLPExporterConfig {
+  logExporter?: CustomExporterConfig;
+  traceExporter?: CustomExporterConfig;
+}
+
+interface IOSConfig {
```

Comment:
is it possible to share this with the definition from packages/core? I'd worry about this getting out of sync between the two

Reply:
please, have a look and let me know what you think